contracts/loot/item/constants.cairo

```diff
@@ -6,54 +6,257 @@
 
 %lang starknet
 
+namespace Material:
+    const generic = 0
+
+    namespace Metal:
+        const generic = 0
+        const ancient = 1
+        const holy = 2
+        const ornate = 3
+        const gold = 4
+        const silver = 5
+        const bronze = 6
+        const platinum = 7
+        const titanium = 8
+        const steel = 9
+    end
+    namespace Cloth:
+        const generic = 0
+        const royal = 1
+        const divine = 2
+        const brightsilk = 3 
+        const silk = 4
+        const wool = 5
+        const linen = 6
+    end
+    namespace Biotic:
```

Comment:
@loothero Any reason we aren't going with your previous idea where demon, dragon, etc are just instances of Organic?

![image](https://user-images.githubusercontent.com/102006890/185946456-8013f1b9-26bf-43e0-9ea3-83029a8d8725.png)